src/components/authenticate/Password/Password.tsx

```diff
@@ -9,9 +9,9 @@ const handleVisibilityToggle = (index: string, state: boolean) => {
   const requirement = document.querySelector(`[data-check="${index}"]`);
 
   if (requirement) {
-    state
-      ? requirement.classList.remove('uk-hidden')
-      : requirement.classList.add('uk-hidden');
+    state ?
```

Comment:
I guess this file wasn't already linted. Interesting. Looks like the changes are all stylistic, so this should be fine